src/GitHub.Exports/Models/RepositoryModel.cs

```diff
@@ -85,7 +67,7 @@ protected set
         public Octicon Icon
```

Comment:
Future: move this out of the model: it's a view model-level reponsibility that only really applies to the repository list in Team Explorer.